content/en/tracing/guide/ingestion_sampling_use_cases.md

```diff
@@ -98,7 +98,9 @@ In addition to head-based sampled traces, you can increase the error sampling ra
 
 {{< img src="/tracing/guide/ingestion_sampling_use_cases/error-spans-sampling.png" alt="Error Sampling" style="width:100%;" >}}
 
-**Note:** Distributed pieces of the trace chunks might not be ingested as the sampling happens locally at the Datadog Agent level.
+**Notes:** 
+- Distributed pieces of the trace chunks might not be ingested as the sampling happens locally at the Datadog Agent level.
+- Starting with **Datadog Agent 6/7.41.0 and higher**, `DD_APM_FEATURES=error_rare_sample_tracer_drop` can be set include spans dropped by tracing library rules or `manual.drop`. More details can be found in the [Error traces section of the Ingestion Mechanisms doc.][1]
```

Comment:
Note to the docs team reviewer: I want to direct users to this section, but I'm not sure the best way to do this:
https://docs-staging.datadoghq.com/wantsui/error-sampler-update/tracing/trace_pipeline/ingestion_mechanisms/?tab=java#error-and-rare-traces

Reply:
Hi @wantsui, I added a new link here that should direct the user to that section. I also made some small grammatical fixes at the same time 👍